factory-package-news/factory-package-news-web.py

```diff
@@ -26,16 +26,21 @@
 import re
 import os
 import sys
+from urlparse import urlparse
 
 import re
 digits_re = re.compile('^[0-9]+$')
 
-_dir = "/var/lib/openqa/factory/snapshot-changes"
+BASE_DIR = '/var/lib'
 
 app = Flask(__name__)
 
+def get_dir(url):
+    return os.path.join(BASE_DIR, urlparse(url).path.lstrip('/'))
```

Comment:
That was the idea with config, as it should only allow for paths in config. I tried some common things like `../` and what not in paths on apache all behaved as expected. The transformation were resolved by apache and the application never even hit just 404.